### PR TITLE
[MMDS][CDAP-13059] add landing page service control for MDMS

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://issues.cask.co/browse/CDAP-13059

TODO: Change the image of experiments

### CODE REVIEW

File: cdap-ui/app/cdap/components/Experiments/ExperimentsServiceControl/index.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+      return (
+        <div className="action-container">
+          <button
+            className="btn btn-primary"
+            onClick={this.enableMMDS}
+            disabled={this.state.loading}
+          >
+            {
+              this.state.loading ?
+                <LoadingSVG height="16px"/>
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/Experiments/ExperimentsServiceControl/index.js]: Inline style? Can we use a css class to this?

**Review by ajainarayanan — COMMENTED** [review]
Just one minor comment. Other 👍 LGTM .. Rebase needed (few list APIs have changed. Might want to check with the general flow)

File: cdap-ui/app/cdap/components/Experiments/ExperimentsServiceControl/index.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    let namespace = getCurrentNamespace();
+
+    MyArtifactApi
+      .list({ namespace })
+      .subscribe(
+        (artifacts) => {
+          let isArtifactPresent = artifacts.find(artifact => artifact.name === MMDSArtifact);
+          if (!isArtifactPresent) {
+            this.setState({
+              artifactAvailable: true
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/Experiments/ExperimentsServiceControl/index.js]: nit: Shouldn't this be `artifactNotAvailable`?

**Review by ajainarayanan — COMMENTED** [review]
👍 LGTM

**Review by bdmogal — APPROVED** [review]

**Pull request merged**